src/sdk/setPrimaryDomain.ts

```diff
@@ -13,10 +13,13 @@ const setPrimaryDomain = async (wallet: Account, domain: string, txParams: TxPar
     const resolverContractInstance = getResolverContract(wallet);
     const invocationScope = resolverContractInstance
       .functions
-      .set_primary({bits: assetId})
+      .set({bits: assetId})
```

Comment:
This call doesn't set the primary domain. You should call `set_primary` function from the registry contract

Reply:
Fixed